dist/vokal-ng-api.js

```diff
@@ -341,10 +355,31 @@ angular.module( "vokal.API", [ "vokal.Humps" ] )
 
                                 return;
                             }
+                            else
+                            {
+                                $rootScope.$broadcast( "APIRequestUnauthorized", options, data, status );
```

Comment:
don't need to return here? 
If this is the same as `that.unauthorizedInterrupt === true` you could do a check for 
```javascript
if ( typeof that.unauthorizedInterrupt === "function" )
{ 
...
}
else 
{
...
}`